src/components/FeeSelector/shared.tsx

```diff
@@ -1,39 +1,40 @@
 import { Trans } from '@lingui/macro'
+import { ChainId, SUPPORTED_CHAINS } from '@uniswap/sdk-core'
 import { FeeAmount } from '@uniswap/v3-sdk'
-import { ALL_SUPPORTED_CHAIN_IDS, SupportedChainId } from 'constants/chains'
 import type { ReactNode } from 'react'
 
 export const FEE_AMOUNT_DETAIL: Record<
   FeeAmount,
-  { label: string; description: ReactNode; supportedChains: SupportedChainId[] }
+  { label: string; description: ReactNode; supportedChains: ChainId[] }
```

Comment:
why not just declare this as `readonly` instead of making a copy of `SUPPORTED_CHAINS`

Reply:
great callout